ci/test/00_setup_env_win64.sh

```diff
@@ -16,4 +16,5 @@ export GOAL="deploy"
 # Prior to 11.0.0, the mingw-w64 headers were missing noreturn attributes, causing warnings when
 # cross-compiling for Windows. https://sourceforge.net/p/mingw-w64/bugs/306/
 # https://github.com/mingw-w64/mingw-w64/commit/1690994f515910a31b9fb7c7bd3a52d4ba987abe
-export BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests CXXFLAGS=-Wno-return-type"
+export BITCOIN_CONFIG="-DREDUCE_EXPORTS=ON -DBUILD_GUI_TESTS=OFF -DCMAKE_CXX_COMPILER=-Wno-return-type \
```

Comment:
This is setting the compiler to be a warning flag, but I don't see any CI failures, so I guess this job isn't being run?

Reply:
> This is setting the compiler to be a warning flag,

Thanks! This typo will be fixed in the next push.

> I don't see any CI failures, so I guess this job isn't being run?

For CI logs, please see [bitcoin#29790](https://github.com/bitcoin/bitcoin/pull/29790) as stated in the PR description.